cypress-io/cypress
# fix(component-testing): avoid cy.viewport failures of invariant

Since we chose to move to TypeScript, the compiled code looks a little different than with babel. We need to be careful with the decorators.

In this case, the inner function to set the callback was not considered an action anymore in TS and was making the second step fail.

## How to test

- run server-ct example
- open the Cell.spec.js file
- you should see the height and width adjust according to this file

closes CT-206
closes CT-212
